app/models/concerns/redis_form.rb

```diff
@@ -50,7 +50,10 @@ def save
 
   def form_matches_schema
     if form.present?
-      errors[:form].concat(JSON::Validator.fully_validate(VetsJsonSchema::SCHEMAS[self.class::FORM_ID], parsed_form))
+      validations = JSON::Validator.fully_validate(VetsJsonSchema::SCHEMAS[self.class::FORM_ID], parsed_form)
+      validations.each do |v|
+        errors.add(:form, v.to_s)
+      end
```

Comment:
Before the method would return `errors[:form]`. Now it's returning the enumerable. I don't know if this will affect anything but here's how you should be able to do what you're asking without doing the gross `v = foo; f.each {}; v` dance.

```suggestion
      JSON::Validator.fully_validate(VetsJsonSchema::SCHEMAS[self.class::FORM_ID], parsed_form).tap do |v|
        errors.add(:form, v.to_s)
      end
```